src/js/tabs/firmware_flasher.js

```diff
@@ -1,6 +1,6 @@
-'use strict';
+import { i18n } from '../localization';
 
-TABS.firmware_flasher = {
+const firmware_flasher = {
```

Comment:
Just for me to understand, do we follow C coding style where the function opening bracket should be on the next line?
https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md